docs/src/main/java/io/helidon/docs/mp/openapi/README.md

```diff
@@ -0,0 +1,17 @@
+To generate a project containing the types which need to be mocked--in case of changes in the upstream generator:
```

Comment:
Do we have an example that generates the exact same sources that we can point to ?
If have single Snippets classes, this can go next to the code in comments.

Reply:
There's no such example.

Reply:
How about this one?
https://github.com/helidon-io/helidon-examples/blob/a1025a8aa30d831175de21bb9df3f6979edb8e83/examples/openapi-tools/quickstart.yaml#L17

Reply:
Different API, different generator output.

Reply:
Seems like a good idea to align the doc and examples.
Can we update this PR to use the example ?

If the example is too basic, we can extend it as needed. We can keep the local yaml file until we can reference the one in the example.

Reply:
Let's consider adding another example based on the Pet Store yaml, keeping the existing example that relates to the familiar QuickStart code.

Reply:
Here is a placeholder issue: https://github.com/helidon-io/helidon-examples/issues/73